docs/using/tsuru.yaml.rst

```diff
@@ -24,7 +24,7 @@ tsuru provides some deployment hooks, like ``restart:before``, ``restart:after``
 and ``build``. Deployment hooks allow developers to run commands before and after
 some commands.
 
-Here is an example about how to declare this hooks in your tsuru.yaml file:
+Here is an example about how to declare these hooks in your tsuru.yaml file:
```

Comment:
Here is an example about how to declare these hooks in your tsuru.yaml file: > "Here is an example ON how to declare these hooks in your tsuru.yaml file:"